cmd/reconciler/main.go

```diff
@@ -122,6 +139,17 @@ var flags = struct {
 	namespaceStrategy: "namespace-strategy",
 }
 
+func timeStringToDuration(t string) (time.Duration, error) {
```

Comment:
nit: name it something more specific given that it expects Duration to be non-negative.